circle.yml

```diff
@@ -1,11 +1,14 @@
 dependencies:
   pre:
-    - npm install hexo-cli mocha -g
+    - npm install hexo-cli mocha bower gulp -g
     - npm install
+    - bower install
   cache_directories:
     - node_modules
+    - themes/trackfellas/bower_components
 
 test:
   override:
+    - gulp
```

Comment:
This should go after bower install in dependencies/pre as it does not imply any test